src/politech_backend/settings.py

```diff
@@ -241,6 +241,16 @@
     'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',)
 }
 
+SWAGGER_SETTINGS = {
```

Comment:
I would have to do a bit more research to confirm, but it seems it needs to be the same as Django. I don't think it can gain knowledge of the endpoint if it can't pass Djangos authentication, unless we add code specifically for Swagger.